c/CMakeLists.txt

```
@@ -47,7 +47,10 @@ if (BUILD_SHARED_LIBS)
   )
   list(APPEND BLAKE3_PKGCONFIG_CFLAGS -DBLAKE3_DLL)
 endif()
-target_include_directories(blake3 PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+target_include_directories(blake3 PUBLIC
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}>
```

```suggestion
  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
```
Less stuff to change if we ever split the `CMakeLists.txt`.